### PR TITLE
[Snippets] Removed limitation on Subgraph creation after Parameters

### PR DESCRIPTION
**Pull request by v-Golubev**
### Details:
 - *Removed limitation on Subgraph creation after Parameters*
 - *Updated tests: layers that were used as plug between Subgraph and Parameters removed from builders*

### Tickets:
 - *90036*

### CODE REVIEW

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/conversion.cpp [hunk 1]
```diff
@@ -52,8 +52,17 @@ class ConvertCPULayerTest : public testing::WithParamInterface<convertLayerTestP
         std::tie(shapes, inPrc, outPrc, cpuParams) = GetParam();
 
         std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
-
-        selectedType = std::string("unknown_") + (inPrc == InferenceEngine::Precision::U8 ? "I8" : inPrc.name());
+        auto primitive = selectedType;
+        if (primitive.empty())
+            primitive = getPrimitiveType();
+        // WA: I32 precision support disabled in snippets => primitive has to be changed
```
Comment (IvanNovoselov) [inline src/plugins/intel_cpu/tests/functional/single_layer_tests/conversion.cpp:58]: Could you please add a todo + ticket number (ideally also add a link in the ticket), so we won't forget to remove this WA when I32 is supported. I believe @eshoguli can give you the ticket id

**Review by IvanNovoselov — APPROVED** [review]
I am so glad we're finally doing this 🥳 Nice job!

File: src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/add.cpp [hunk 2]
```diff
@@ -17,49 +17,28 @@ INSTANTIATE_TEST_SUITE_P(smoke_Snippets_Eltwise, Add,
...
-        AddSinhConst::getTestCaseName);
-
 INSTANTIATE_TEST_SUITE_P(smoke_Snippets_Eltwise, AddRollConst,
         ::testing::Combine(
                 ::testing::Values(ov::Shape {1, 42, 16, 64}),
                 ::testing::Values(ov::element::f32),
-                ::testing::Values(2), // Add + roll after inputs
-                ::testing::Values(1), // Subgraph is created, since the inputs are followed by converts
+                ::testing::Values(2),
+                ::testing::Values(1),
```
[the remark this answers is not in the document]
Comment (v-Golubev) [reply src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/add.cpp]: These comments were removed by mistake, thanks for your correction.

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/conversion.cpp (hunk 1 above)
Comment (v-Golubev) [reply src/plugins/intel_cpu/tests/functional/single_layer_tests/conversion.cpp:58]: Todo and ticket number are added. However, we will have to remove this WA anyway because these tests will fail when I32 precision is enabled for snippets

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/activation.cpp [hunk 3]
```diff
@@ -154,9 +154,8 @@ std::vector<Precision> netPrc = {
         Precision::FP32
 };
 
-/* ============= Activation (1D) ============= */
```
Comment (a-sidorova) [inline src/plugins/intel_cpu/tests/functional/single_layer_tests/activation.cpp:157]: FYI
As I remember, `1D` means one spatial dimension - `w` (`ncw`). So previous header was correct as well 😃

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/activation.cpp [hunk 4]
```diff
@@ -178,7 +177,28 @@ const auto basicCases3D = ::testing::Combine(
 
 INSTANTIATE_TEST_SUITE_P(smoke_Activation3D_Eltwise_CPU_BF16, ActivationLayerCPUTest, basicCases3D, ActivationLayerCPUTest::getTestCaseName);
 
-/* ============= Activation (2D) ============= */
+const std::map<ActivationTypes, std::vector<std::vector<float>>> activationTypes_blocked = {
+        {Mish,        {{}}},
+        {SoftSign,    {{}}}
```
Comment (a-sidorova) [inline src/plugins/intel_cpu/tests/functional/single_layer_tests/activation.cpp:182]: I think we should create ticket to add these ops to snippets. It should be in another PR, sure. Could you please create the ticket? Thanks!

**Review by a-sidorova — APPROVED** [review]

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/activation.cpp (hunk 4 above)
Comment (v-Golubev) [reply src/plugins/intel_cpu/tests/functional/single_layer_tests/activation.cpp:182]: Good point, thanks. 99811 and 99812 are created

File: src/plugins/intel_cpu/tests/functional/single_layer_tests/activation.cpp (hunk 3 above)
Comment (v-Golubev) [reply src/plugins/intel_cpu/tests/functional/single_layer_tests/activation.cpp:157]: Thank you for the clarification) I will revert my changes

**Review by dmitry-gorokhov — APPROVED** [review]
